docs/content/components/collection/gridlist/selectlist-multiple-selection.demo.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+import { Button, SelectList } from '@marigold/components';
+
+let pokemons = [
+  { id: 'charizard', name: 'Charizard' },
+  { id: 'blastoise', name: 'Blastoise' },
+  { id: 'venusaur', name: 'Venusaur' },
+  { id: 'pikachu', name: 'Pikachu' },
+];
+
+export default () => {
+  const [selectedPokemons, setSelectedPokemons] = useState(['charizard']);
+  return (
+    <>
+      <SelectList
+        aria-label="Favorite pokemon"
+        selectionMode="multiple"
+        items={pokemons}
+        selectedKeys={selectedPokemons}
+        onChange={setSelectedPokemons}
+      >
+        {(item: { id: any; name: string }) => (
+          <SelectList.Item id={item.id}>
+            {item.name}
+            <Button
+              aria-label="Info"
+              onPress={() => alert(`Info for ${item.name}...`)}
+              className="ml-auto"
+            >
+              ⓘ
```

Comment:
don't we have an own icon for it?

Reply:
@sarahgm   i added a new one, check it plz